client/src/server_messages.zig

```diff
@@ -1,124 +1,141 @@
-const erl = @import("erlang/config.zig");
+const erl = @import("erlang.zig");
 const std = @import("std");
 const rl = @import("raylib");
-const sender = @import("erlang/sender.zig");
-const receiver = @import("erlang/receiver.zig");
-
-pub const Action = enum {
-    user_registry,
-    user_login,
-    character_list,
-    debug,
-};
 
-pub const Erlang_Response = union(enum) {
-    ok: void,
-    @"error": [:0]const u8,
+fn createAnonymousStruct(comptime T: type, comptime keys: []const [:0]const u8) type {
```

Comment:
We're not interoping with C. We don't need the slice to be null-terminated.

Reply:
The entire refactoring of null-terminated strings will be addressed later on.